README.md

```diff
@@ -1,60 +1,81 @@
-DAGMC-viz Documentation
-=======================
+# DAGMC-viz Documentation
+The purpose of this toolkit is to assist with the visualization of DAGMC geometries in VisIt. This can include creating a default interactive view in VisIt, removing the graveyard volume from a given geometry, and expanding vector tags on a given mesh to facilitate database animation.
 
-The Direct Accelerated Geometry Monte Carlo visualization toolkit requires Python 2.7, VisIt, and MOAB. The inclusion of VisIt and MOAB in the PYTHONPATH and the VisIt executable in the PATH is required. A user can input a geometry file and a data file to produce default interactive output in VisIt, with the option to save images of the plot windows as well. Functionality allowing the user to have full control of determining the output will be implemented in the future.
+## Installation
 
-----------------------------------------
+The Direct Accelerated Geometry Monte Carlo visualization toolkit requires Python 2.7, VisIt 2.3.13 or above, and MOAB 5.1.0. The inclusion of VisIt and MOAB in the PYTHONPATH and the VisIt and MOAB executables in the PATH are required.The toolkit can be installed on a linux platform with the following command.
+
+```
+pip install -i https://test.pypi.org/simple/dagmc-viz
+```
+
+If the user is in a Python environment, each of the three submodules can be accessed in the following way.
+
+```
+import svalinn_tools.{SUBMODULE}
+```
+
+## Description
 
-1. DataLoading.py can be used to create a default output in VisIt from an h5m geometry file and an h5m or vtk data file. The following four interactive plot windows are displayed in a 2x2 grid.
+### data_loading.py 
```

Comment:
Might be better to give these headings based on what they do:
```suggestion
### Data Loading
```
(and similarly with the others)